DataMerge.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>You could maybe add some pointers to the relevant notebooks here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>